crates/api_models/src/payouts.rs

```diff
@@ -163,37 +163,66 @@ pub struct Card {
 }
 
 #[cfg(feature = "payouts")]
-#[derive(Eq, PartialEq, Clone, Debug, Serialize, ToSchema)]
-/// TODO: Implement standard format display for Bank
-pub struct Bank {
+#[derive(Eq, PartialEq, Clone, Debug, Deserialize, Serialize, ToSchema)]
+#[serde(rename_all = "lowercase")]
+pub enum Bank {
+    Ach(AchBankTransfer),
+    Bacs(BacsBankTransfer),
+    Sepa(SepaBankTransfer),
+}
+
+#[cfg(feature = "payouts")]
+#[derive(Default, Eq, PartialEq, Clone, Debug, Deserialize, Serialize, ToSchema)]
+pub struct AchBankTransfer {
+    /// Bank name
+    #[schema(value_type = String, example = "Deutsche Bank")]
+    pub bank_name: String,
+
+    /// Bank country code
+    #[schema(value_type = String, example = "US")]
+    pub bank_country_code: api_enums::CountryAlpha2,
+
+    /// Bank city
+    #[schema(value_type = String, example = "California")]
+    pub bank_city: String,
+
     /// Bank account number is an unique identifier assigned by a bank to a customer.
     #[schema(value_type = String, example = "000123456")]
-    pub bank_account_number: Option<String>,
+    pub bank_account_number: String,
 
     /// [9 digits] Routing number - used in USA for identifying a specific bank.
     #[schema(value_type = String, example = "110000000")]
-    pub bank_routing_number: Option<String>,
+    pub bank_routing_number: String,
+}
 
-    /// International Bank Account Number (iban) - used in many countries for identifying a bank along with it's customer.
-    #[schema(value_type = String, example = "DE89370400440532013000")]
-    pub iban: Option<String>,
+#[cfg(feature = "payouts")]
+#[derive(Default, Eq, PartialEq, Clone, Debug, Deserialize, Serialize, ToSchema)]
+pub struct BacsBankTransfer {
+    /// Bank name
+    #[schema(value_type = String, example = "Deutsche Bank")]
+    pub bank_name: String,
```

Comment:
mask it with secret

Reply:
Resolved in a separate commit - https://github.com/juspay/hyperswitch/pull/1384/commits/ac41daafceda596d06c428eb4b15eb15fadd337b